change/@fluentui-react-field-26ccf66f-7c26-4033-be6d-f975081fe7b4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 20 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26352&commitId=22aa1884bcbc8d6dee1b0e7d8e6200bb0594ace0&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1753 screenshots unchanged, 0 screenshots with different dimensions and 20 screenshots with visible difference.


  <details><summary><b>unknown</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CheckboxField converged.horizontal.default.chromium.png|564| Changed|
|CheckboxField converged.validation.default.chromium.png|3342| Changed|
|ComboboxField converged.horizontal.default.chromium.png|803| Changed|
|ComboboxField converged.validation.default.chromium.png|4723| Changed|
|InputField converged.horizontal.default.chromium.png|803| Changed|
|InputField converged.validation.default.chromium.png|4444| Changed|
|ProgressField converged.horizontal.default.chromium.png|803| Changed|
|ProgressField converged.validation.default.chromium.png|5292| Changed|
|RadioGroupField converged.horizontal.default.chromium.png|1301| Changed|
|RadioGroupField converged.validation.default.chromium.png|4923| Changed|
|SelectField converged.horizontal.default.chromium.png|803| Changed|
|SelectField converged.validation.default.chromium.png|4578| Changed|
|SliderField converged.horizontal.default.chromium.png|803| Changed|
|SliderField converged.validation.default.chromium.png|4230| Changed|
|SpinButtonField converged.horizontal.default.chromium.png|803| Changed|
|SpinButtonField converged.validation.default.chromium.png|4610| Changed|
|SwitchField converged.horizontal.default.chromium.png|803| Changed|
|SwitchField converged.validation.default.chromium.png|3854| Changed|
|TextareaField converged.horizontal.default.chromium.png|1301| Changed|
|TextareaField converged.validation.default.chromium.png|4472| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
There are three basic changes that are by design:
1. ✅ The label in horizontal layout reflowed. This is caused by fixing the margin between the text and input (it was previously not getting applied). The new margin is correct.
2. ✅ The icons are 1px higher in the line with the text. This more closely matches the design than the previous state.
3. ✅ The icon for 'custom status' is slightly lighter. I can't really explain this one, but it seems like the new behavior is correct.